packages/scandipwa/src/component/Router/Router.component.tsx

```diff
@@ -58,6 +58,7 @@ export const PasswordChangePage = lazy(() => import(/* webpackMode: "lazy", webp
 export const SearchPage = lazy(() => import(/* webpackMode: "lazy", webpackChunkName: "search" */ 'Route/SearchPage'));
 export const SendConfirmationPage = lazy(() => import(/* webpackMode: "lazy", webpackChunkName: "cms" */ 'Route/SendConfirmationPage'));
 export const ConfirmAccountPage = lazy(() => import(/* webpackMode: "lazy", webpackChunkName: "cms" */ 'Route/ConfirmAccountPage'));
+export const ConfirmNewsletterPage = lazy(() => import(/* webpackMode: "lazy", webpackChunkName: "cms" */ 'Route/ConfirmNewsletterPage'));
```

Comment:
Good if we will change webpackChunkName for them. They are definitely shouldn't go into cms chunk